.github/workflows/ci-nix.yaml

```diff
@@ -253,7 +253,7 @@ jobs:
 
     - name: ❄ Nix Flake Check
       run: |
-        nix flake check -L
+        nix --option sandbox false flake check -L
```

Comment:
This is currently failing in CI: https://github.com/cardano-scaling/hydra/actions/runs/10960957119/job/30436784844

Reply:
Fixed